packages/vega-typings/types/runtime/dataflow.d.ts

```diff
@@ -0,0 +1,28 @@
+export interface Changeset {
+  insert(tuples: any): this;
```

Comment:
Should this be an array?

Reply:
The input can be an array of tuples, or a single tuple.